systems/citybuilding.go

```diff
@@ -4,12 +4,14 @@ import (
 	"engo.io/ecs"
 	"engo.io/engo"
 	"engo.io/engo/common"
+
+	"fmt"
```

Comment:
gofmt likes the stdlib packages to be at the top separated by a new line with external packages.